lib/do_it/repo.ex

```diff
@@ -20,4 +21,8 @@ defmodule DoIt.Repo do
     List.changeset(list, %{title: title})
     |> update()
   end
+
+  def create_todo(params) do
+    Todo.create_changeset(params) |> insert()
```

Comment:
```suggestion
    params
    |> Todo.create_changeset()
    |> insert()
```

I personally find it easier to read when pipelines start with the "raw" value.

Could potentially be overwhelming at the start, so I'm not sure whether it's a proper "suggestion", but you could have a look at adding [Credo](https://github.com/rrrene/credo) to your project 💅 
It's a linter for Elixir and can point you towards loads of useful best practice 🔍

Reply:
I think I read about Credo somewhere but then forgot about it. Will check it out! I often wonder what would be the best practice and if I can get some help there right away, that will be awesome. Thanks for the suggestion 👍